### PR TITLE
Refactor layouts

### PR DESCRIPTION
**Pull request by mejdoo**
## What's new in this PR?

- Refactor some layouts to respect the coding style.

## Note

Those changes are made in March before my sick leave as part of Kotlin Code review. I rebased to develop and fixed the conflicts.

#### APK
[Download build #1967](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1967/artifact/build/artifact/wire-dev-PR2792-1967.apk)
[Download build #2091](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2091/artifact/build/artifact/wire-dev-PR2792-2091.apk)

### CODE REVIEW

File: app/src/main/res-migration/layout/activity_settings_support.xml [hunk 1]
```diff
@@ -26,26 +26,22 @@
     android:orientation="vertical">
```
Comment (gizemb) [inline app/src/main/res-migration/layout/activity_settings_support.xml:26]: seems like this 
```
<ConstraintLayout>
   <Toolbar />
   <FrameLayout />
</ConstraintLayout>
```
is used in many screens. maybe we can re-use a single layout? what do you think?

in a separate commit of course :)

**Comment by mejdoo** [comment]
@gizemb those styles are only used in the welcome screen. I remember that that Victor reinvented something new. We need a larger discussion with design team regarding the styleguide and components for Android. I didn't even get any feedback about color naming. I will push again.

File: app/src/main/res-migration/layout/activity_settings_support.xml (hunk 1 above)
Comment (mejdoo) [reply app/src/main/res-migration/layout/activity_settings_support.xml:26]: yes, I think we can uses a single layout by different activities as we have the same template. Will do later in a an other PR.

**Comment by mejdoo** [comment]
build again

**Comment by mejdoo** [comment]
build again

**Comment by mejdoo** [comment]
build again

**Pull request merged**